Sources/App/configure.swift

```diff
@@ -18,7 +16,7 @@ public func configure(_ config: inout Config, _ env: inout Environment, _ servic
     // Configure MySQL
     // TODO: change later to support environments
     let mysqlDatabaseConfig = MySQLDatabaseConfig(hostname: "127.0.0.1",
-                                                  port: 32773,
+                                                  port: 32768,
                                                   username: "root",
                                                   password: "youclap-sql-pw",
                                                   database: "youclap-dev",
```

Comment:
All these parameters should be a variable, introduced as an environment variable, right?

Reply:
* variables

Reply:
this is just an example, so it's ok to leave the credentials here. those are not even correct.

regarding the inject them as env vars, yes, i'm doing that in marmoto